### DISCUSSION TITLE
Emoji's in WYSIWYG

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#11727] [Archive]
Wouldn't it be cool if we could include the ability to use emoji's in the WYSIWYG editor?

If I were to up the 😊 emoji into the editor and click save rather than saving the encoding for the emoji it just saves ????

Could we possibly add support for emoji's? Pleeeease

**Comment by rijkvanzanten** [comment]
> If I were to up the 😊 emoji into the editor and click save rather than saving the encoding for the emoji it just saves ????


This is based on the database and database-charset/collation you're using. From the `????`, I'm assuming you're using MySQL with a non utf-8 character set

**Reply by ghost** [reply]
@rijkvanzanten That would make cense haha

**Comment by joselcvarela** [comment]
Hope this issue can help you:
https://github.com/directus/directus/issues/8561
